packages/expo-google-sign-in/ios/EXGoogleSignIn/EXAuthTask.h

```diff
@@ -0,0 +1,19 @@
+//
+//  EXAuthTask.h
+//  EXGoogleSignIn
+//
+//  Created by Evan Bacon on 10/19/18.
+//
+
+#import <Foundation/Foundation.h>
+#import <EXCore/EXModuleRegistry.h>
+
+static NSString *const E_CONCURRENT_TASK_IN_PROGRESS = @"E_CONCURRENT_TASK_IN_PROGRESS";
```

Comment:
If we want to make this module versioning-compatible we'll need to prefix those variables with `EX`. You never know what will happen to your module some day! 😉